manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Bonnefon.glogg
+PackageVersion: 1.1.4
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://glogg.bonnefon.org/files/glogg-v1.1.4-x86_64-setup.exe
+  InstallerSha256: 08FE13B713327BEF93298B6D11717A93BBAA9D49165995BE93F4A3282E76B22F

```

```suggestion
  InstallerSha256: 08FE13B713327BEF93298B6D11717A93BBAA9D49165995BE93F4A3282E76B22F
  AppsAndFeaturesEntries:
    - DisplayVersion: v1.1.4-x86_64
      Publisher:
      ProductCode: glogg
```
For 1:1 match with Registry values that helps with better upgrade matching